src/sagemaker/estimator.py

```diff
@@ -1280,10 +1280,11 @@ def __init__(
                 (default: logging.INFO). Valid values are defined in the Python
                 logging module.
             code_location (str): The S3 prefix URI where custom code will be
-                uploaded (default: None). The code file uploaded in S3 is
-                'code_location/source/sourcedir.tar.gz'. If not specified, the
-                default code location is s3://default_bucket/job-name/. And code
-                file uploaded to S3 is
+                uploaded (default: None) - don't include a trailing slash since 
+                a string prepended with / is appended to code_location. The code 
+                file uploaded in S3 is 'code_location/job-name/source/sourcedir.tar.gz'.
+                If not specified, the default code location is s3://default_bucket/job-name/.
+                And code file uploaded to S3 is
                 s3://default_bucket/job-name/source/sourcedir.tar.gz
```

Comment:
You can remove this sentence - it's redundant with "The code file uploaded in S3 is 'code_location/job-name/source/sourcedir.tar.gz'" above